MigrationTool/Patch_v2_11_1.cs

```diff
@@ -0,0 +1,45 @@
+﻿using Rampastring.Tools;
+
+namespace MigrationTool;
+
+internal class Patch_v2_11_1 : Patch
+{
+    public Patch_v2_11_1 (string clientPath) : base (clientPath)
+    {
+        ClientVersion = Version.v2_11_1;
+    }
+
+    public override Patch Apply()
+    {
+        base.Apply();
+
+        // Add ClientDefinitions.ini->[Settings]->RecommendedResolutions, MaximumRenderWidth, MaximumRenderHeight
+        IniFile clientDefsIni = new IniFile(SafePath.CombineFilePath(ResouresDir.FullName, "ClientDefinitions.ini"));
+        AddKeyWithLog(clientDefsIni, "Settings", "MaximumRenderWidth", "1280");
+        AddKeyWithLog(clientDefsIni, "Settings", "MaximumRenderHeight", "720");
+        var width = clientDefsIni.GetStringValue("Settings", "MaximumRenderWidth", "1280");
+        var height = clientDefsIni.GetStringValue("Settings", "MaximumRenderHeight", "720");
+        AddKeyWithLog(clientDefsIni, "Settings", "RecommendedResolutions", $"{width}x{height}");
+        clientDefsIni.WriteIniFile();
+
+        // Rename GameLobbyBase.ini->[SkirmishLobby]->BtnSaveLoadGameOptions to btnSaveLoadGameOptions
+        Logger.Log("Update name from BtnSaveLoadGameOptions to btnSaveLoadGameOptions in GameLobbyBase.ini->[SkirmishLobby]");
+        IniFile glb = new IniFile(SafePath.CombineFilePath(ResouresDir.FullName, "GameLobbyBase.ini"));
+        var presets = glb.GetSection("BtnSaveLoadGameOptions");
+
+        if (presets == null)
+            return this;
+
+        presets.SectionName = "btnSaveLoadGameOptions";
+
+        foreach (var pair in glb.GetSection("SkirmishLobby").Keys)
+        {
+            if (pair.Value.Contains("BtnSaveLoadGameOptions"))
+                pair.Value.Replace("BtnSaveLoadGameOptions", "btnSaveLoadGameOptions");
```

Comment:
CA1806 Apply calls Replace but does not use the new string instance that the method returns. Pass the instance as an argument to another method, assign the instance to a variable, or remove the call if it is unnecessary.